internal/jobobject/jobobject.go

```diff
@@ -58,31 +58,52 @@ var (
 	ErrNotRegistered = errors.New("job is not registered to receive notifications")
 )
 
+type OptionsJobObject struct {
```

Comment:
Just `Options` seems fine to me as it's in the job object package but let me know what you think. We already changed to just `Create` for creating them to avoid the redundancy of `jobobject.CreateJobObject` so this seems like it fits the same bill

Reply:
I just did this to follow the naming used for lcow and wcow options structs. I'm not sure which would be preferred.